src/ape/api/accounts.py

```diff
@@ -135,9 +141,17 @@ def transfer(
             txn.data = self._convert(data, bytes)
 
         if value:
+            if "send_everything" in kwargs and kwargs["send_everything"]:
```

Comment:
this can just be 
```suggestion
            if not kwargs.get("send_everything"):
```

Reply:
```
FAILED tests/functional/test_accounts.py::test_transfer - ValueError: Kwarg send_everything=True requires transfer without value argument
FAILED tests/functional/test_accounts.py::test_transfer_with_value_send_everything_true - Failed: DID NOT RAISE <class 'ValueError'>
FAILED tests/functional/test_accounts.py::test_transfer_using_type_0 - ValueError: Kwarg send_everything=True requires transfer without value argument
FAILED tests/functional/test_accounts.py::test_send_transaction_with_bad_nonce - ValueError: Kwarg send_everything=True requires transfer without value argument
FAILED tests/functional/test_accounts.py::test_send_transaction_without_enough_funds - ValueError: Kwarg send_everything=True requires transfer without value argument
FAILED tests/functional/test_accounts.py::test_send_transaction_sets_defaults - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_sign_message_with_prompts - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_transfer_with_prompts - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_autosign_messages - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_autosign_transactions - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_unlock_with_passphrase_and_sign_message - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_unlock_from_prompt_and_sign_message - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_unlock_with_passphrase_and_sign_transaction - ValueError: Kwarg send_everything=True requires transfer without value argument
ERROR tests/functional/test_accounts.py::test_unlock_from_prompt_and_sign_transaction - ValueError: Kwarg send_everything=True requires transfer without value argument
```

Reply:
Oh I see, I was reading the condition wrong